src/tblib/__init__.py

```diff
@@ -19,22 +19,16 @@
 
 
 class _AttrDict(dict):
+    __slots__ = ()
+
     def __getattr__(self, attr):
         return self[attr]
 
 
-class __traceback_maker(Exception):
```

Comment:
Did this custom exception create a problem for you or what warranted its removal?